cmake/TestImpactFramework/LYTestImpactFramework.cmake

```diff
@@ -439,6 +441,45 @@ function(ly_test_impact_write_config_file CONFIG_TEMPLATE_FILE BIN_DIR)
     message(DEBUG "Test impact framework post steps complete")
 endfunction()
 
+function(ly_test_impact_write_pytest_file)
+    set(config_array "")
+    foreach(config_type ${LY_CONFIGURATION_TYPES})
```

Comment:
These `foreach`s could be wrapped up in a helper func that accepts a list of values and then compiles each entry and returns a comma separated string of said compiled values.